### PR TITLE
[env/burn-staging] Update show only events in that venue by default 

### PR DESCRIPTION
**Pull request by ermiaSparkle**
Resolve issue:
- https://github.com/sparkletown/internal-sparkle-issues/issues/1002#issuecomment-911915629

### CODE REVIEW

**Review by denisdimitrov — APPROVED** [review]
LGTM!

File: src/components/organisms/NavBarSchedule/NavBarSchedule.tsx [hunk 1]
```diff
@@ -94,7 +94,9 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
     currentVenueId: venueId,
   });
 
-  const [filterRelatedEvents, setFilterRelatedEvents] = useState(false);
+  const [filterRelatedEvents, setFilterRelatedEvents] = useState(
+    currentVenue?.id !== sovereignVenue?.id
+  );
```
Comment (goran-peoski-work) [inline src/components/organisms/NavBarSchedule/NavBarSchedule.tsx:99]: What happens if both are `undefined` ?

Comment (ermiaSparkle) [reply src/components/organisms/NavBarSchedule/NavBarSchedule.tsx:99]: will be `false` and it will work as normal.

Comment (goran-peoski-work) [reply src/components/organisms/NavBarSchedule/NavBarSchedule.tsx:99]: K, good enough. 

I've been making the venue/venueId optional for the purpose of 404 inside admin, just so there is NavBar visible

**Pull request merged**